bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```
@@ -500,92 +501,185 @@
+            baseZoom = zoomWithGap;
+          }
+
+          const adjustedXPos = xOffset;
+          const adjustedYPos = yOffset;
+
+          throwIfInvalid(adjustedXPos, 'camera.x');
+          throwIfInvalid(adjustedYPos, 'camera.y');
+
+          const camera = tlEditorRef.current.getCamera();
+          const formattedPageId = Number(curPageIdRef.current);
+          if (isNaN(formattedPageId)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)